src/node_constants.cc

```diff
@@ -795,11 +795,39 @@ void DefinePriorityConstants(Local<Object> target) {
 #endif
 }
 
+// Convert nid's to the string representation of their OID. Non-reentrant, and
+// will abort if called with invalid nids (so only pass values from OpenSSL's
+// headers).
+static const char* OBJ_nid2oid(int nid) {
```

Comment:
This more or less duplicates `OBJ_nid2ln()`, doesn't it?

(Or maybe `OBJ_nid2ln()` followed by `OBJ_nid2sn()` if the former doesn't find anything.)

Reply:
No, check out `nid_objs` in `deps/openssl/openssl/crypto/objects/obj_dat.h`. I'd hoped the ln would be what I want, but it's the name used in the ASN.1 (as opposed to the sn which is the openssl abbreviated name). Its the "txt" I want, and that can only be gotten this way (AFAICT).

Reply:
Oh, I get it now. Okay, objection withdrawn but can you use `CHECK_NOT_NULL(obj)`?